### PR TITLE
NONE: get rid of maybe jwt something something

### PR DESCRIPTION
**Pull request by bgvozdev**
**What's in this PR?**
Removing some remnants of jiraHost/jwt query params

**Why**
Cause they are not secure

**Added feature flags**
Nah

**How has this been tested?**  
Locally

### CODE REVIEW

**Review by joshkay10 — APPROVED** [review]

File: static/js/github-create-branch-options.js [hunk 1]
```diff
@@ -45,11 +45,9 @@ $(document).ready(() => {
 });
 
 const createUrlForGH = (uuid, multiGHInstance) => {
-	const jiraHost = $("#jiraHost").val();
-	const jwt = $("#jwt").val();
   return uuid ?
-    `github/${uuid}/create-branch?issueKey=${issueKey}&issueSummary=${issueSummary}&jiraHost=${jiraHost}&jwt=${jwt}&ghRedirect=to&multiGHInstance=${multiGHInstance}` :
-    `github/create-branch?issueKey=${issueKey}&issueSummary=${issueSummary}&jiraHost=${jiraHost}&jwt=${jwt}&multiGHInstance=${multiGHInstance}`;
+    `github/${uuid}/create-branch?issueKey=${issueKey}&issueSummary=${issueSummary}&ghRedirect=to&multiGHInstance=${multiGHInstance}` :
```
Comment (gxueatlassian) [inline static/js/github-create-branch-options.js:49]: Did you tested it that it still works without `jwt=${jwt}`

Comment (joshkay10) [inline static/js/github-create-branch-options.js:49]: im not sure if JWT still needs to be passed through here, will have a check

Comment (bgvozdev) [reply static/js/github-create-branch-options.js:49]: Yep. jiraHost is put to session by jwtSymmetricMiddleware and is used from there afterwards

**Review by gxueatlassian — APPROVED** [review]

File: static/js/github-create-branch-options.js (hunk 1 above)
Comment (joshkay10) [reply static/js/github-create-branch-options.js:49]: nope, looks like used locals values now, should be ok

Comment (bgvozdev) [reply static/js/github-create-branch-options.js:49]: I don't think so: once JWT is verified it is put to session and used from there after that https://github.com/atlassian/github-for-jira/blob/e698f8241b16668bd2ec659737061a4ce1b552fd/src/middleware/jira-symmetric-jwt-middleware.ts#L45-L45

**Pull request merged**